www/index.html

```diff
@@ -39,26 +40,32 @@
 				<div id="verticalCenter">
 					<H1 class="mainHeader"><a onclick="gotoMainPage();" class="mainHeaderLink">YodaQA Live</a></H1>
 					<!--Form for asking Yoda-->
-					<form method="post" action="http://qa.ailao.eu/q" id="ask">
 						<div class="ui-grid-b" style="margin-bottom: -10px;">
 							<div class="ui-block-a" style="width:93%" id="searchBlock">
 								<input type="search" name="text" id="search" placeholder="Who wrote Ender's Game?"
 								       required>
+								<!--<div id="artificialConceptsBlock">
+									<input type="hidden" name="numberOfConcepts" id="numberOfConcepts" value="1">
+									<br>
+									<input type="text" name="fullLabel1" id="fullLabel1" placeholder="Concept Label" >
+									<input type="number" name="pageID1" id="pageID1" placeholder="Page id">
+								</div>
+								<div id="artificialConceptButtonBlock">
+									<input type="button" name="moreConceptButton" id="moreConceptButton" value="Add concept">
+								</div>-->
```

Comment:
Sorry, I missed this. It was remain of older version. I deleted it and also deleted unused methods in artificialConceptHandler.js, which were connected to this part of code.